code/modules/projectiles/guns/projectile/automatic/drozd.dm

```diff
@@ -1,22 +1,26 @@
 /obj/item/weapon/gun/projectile/automatic/drozd
 	name = "Excelsior .35 Auto \"Drozd\""
-	desc = "An excellent fully automatic submachinegun. Famous for it's perfomance in close quarters. Uses .35 Auto rounds."
+	desc = "An excellent fully automatic submachinegun. Made by \"Excelsior\" by reverse-engineering and improving upon Moebius \
```

Comment:
"An excellent fully automatic submachine gun, Made by \"Excelsior\", reverse engineered from the \"Moebius"\ C20r, with an emphasis on armor penetration over direct damage. Uses .35 Auto rounds."

Reply:
Thing is Moebius is also reverse-engineered C20r, so it's reverse-engineered from reverse-engineered kind of joke. We already have similar joke in AK description.

Reply:
Replaced my mess with this.